src/storage/redis_db.cc

```diff
@@ -155,6 +158,7 @@ rocksdb::Status Database::TTL(const Slice &user_key, int *ttl) {
   Metadata metadata(kRedisNone, false);
   metadata.Decode(value);
```

Comment:
When Decode happens here, would it have the right data when a 32bit size instance read 64bits data?

Reply:
oops, seems `Decode` reset it flags rather than default argument. Seems ok to me